test/CNStakedKLAY.ts

```diff
@@ -568,6 +576,21 @@ describe('CNStakedKLAY', () => {
         });
       });
 
+      describe('#unstakeAll()', () => {
+        it('should unstake all balances', async () => {
+          await cnStakedKLAY.for.alice.unstakeAll();
+
+          await expectBalanceOf(accounts.alice).to.equal(0n);
+        });
+
+        it('should not be able to unstake all if balance is 0', async () => {
```

Comment:
can we also cover a case where `share` is not zero, but `balance` is zero?

Reply:
it looks like difficult to cause precision errors,, 🤔